pkg/raft/testdata/snapshot_succeed_via_app_resp.txt

```diff
@@ -60,7 +60,7 @@ INFO newRaft 3 [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastter
 # again.
 tick-heartbeat 1
 ----
-ok
+DEBUG ignore sending snapshot to 3 since it is not recently active
```

Comment:
Looks weird. We've constructed a snapshot (probably because of the `compact` we weren't able to construct a `MsgApp`) and were trying to send it, but dropped. It would be good to not construct it in the first place then :) Consider leaving a TODO, don't need to get to the bottom of it now.